### DISCUSSION TITLE
How to add a readme file for the account page, not a repository

### DISCUSSION THREAD

**Discussion by controllerandmouse** [community/community#65716] [New to GitHub]
I'm looking for guidance on how to add a README file to my GitHub account page, separate from repository-specific READMEs. I want to create a personalized introduction and share additional information about my skills, interests, and contributions on my GitHub account's main page. Any help or insights on achieving this would be greatly appreciated!

**Comment by jurecapuder** [comment]
Hello,

To add a personalized README file to your GitHub account page, you can follow these steps:

1. **Create a New Repository**: Start by creating a new GitHub repository. This repository will serve as a placeholder for your README file. You can make it public or private, depending on your preference.

2. **Name Your Repository**: Choose a name for your repository. A common convention is to use your GitHub username, like "your-username.github.io," but you can choose any name you like.

3. **Initialize with a README**: During the repository creation process, GitHub provides an option to initialize the repository with a README file. Make sure to check this option. It will create an initial README file for you.

4. **Edit Your README**: After the repository is created, navigate to it on GitHub. You'll see the README file you initialized in the previous step. Click on it to open it.

5. **Edit the README**: You can now edit the README file to include your personalized introduction, information about your skills, interests, contributions, and any other details you want to share. GitHub supports Markdown, so you can format your content as needed.

6. **Commit Changes**: Once you've made your changes to the README, scroll down to the bottom of the page, where you'll find a section to commit your changes. Provide a commit message (e.g., "Updated README with personal info") and click the "Commit changes" button.

7. **Set Repository as Your User or Organization Page**: To ensure that this repository is used as your GitHub account's main page, go to the "Settings" tab of the repository. Under the "GitHub Pages" section, configure the source branch to be the main branch (usually "main" or "master"). GitHub will build your user page from this repository.

8. **Access Your Profile**: Finally, you can access your personalized GitHub account page by going to `https://github.com/your-username` (replace "your-username" with your actual GitHub username). Your README content will be displayed on your account's main page.

Remember that it might take a few minutes for changes to reflect on your GitHub account page, especially if it's the first time you're setting this up.

With these steps, you can create a custom README to showcase your profile and provide additional context about yourself on your GitHub account's main page.

**Reply by controllerandmouse** [reply]
Thank you that solves my problem!

**Comment by iskanderisker** [comment]
Thanks login this page Account tiktok please help me

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
